### PR TITLE
Feature/moov 2615 support lightning attempts

### PR DESCRIPTION
**Pull request by arifmatin**
image

image

### CODE REVIEW

File: packages/components/src/components/ui/atoms/Icon/Icon.tsx [hunk 1]
```diff
@@ -356,6 +356,121 @@ export const Icons = {
...
+        strokeLinejoin="round"
+      />
+      <circle cx="22" cy="8" r="6" fill="#29A37A" />
+      <path
+        d="M25.1667 5.66667L20.8333 10L19 8.16667"
+        stroke="white"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </SVG>
```
Comment (Publicker) [inline packages/components/src/components/ui/atoms/Icon/Icon.tsx:472]: If possible, can we remove the strokes colors/linecap/linejoin from the SVG itself and set the color from where it's used?

Comment (arifmatin) [reply packages/components/src/components/ui/atoms/Icon/Icon.tsx:472]: Ah yes will do

**Review by pabloi-nf — CHANGES_REQUESTED** [review]
- The icon is getting a stroke around the green circle that shouldn't be there 
image
- When I read "1 Bitcoin payment received" it seemd I was receiving 1 BTC as a payment. Would it be correct and less misleading to say "1 Lightning payment received"?

**Review by agranillonf — APPROVED** [review]
Same question as Pablo I. regarding the "Bitcoin payment" labeling.

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/182.

**Comment by arifmatin** [comment]
image

image

**Review by pabloi-nf — APPROVED** [review]

File: packages/components/src/utils/parsers.ts [hunk 2]
```diff
@@ -360,6 +360,33 @@ const remotePaymentRequestToLocalPaymentRequest = (
...
+
+      if (paymentAttempt.settleFailedAt) {
+        events.push({
+          eventType: LocalPaymentAttemptEventType.InvoiceExpired,
+          occurredAt: new Date(paymentAttempt.settleFailedAt),
+          currency: paymentAttempt.currency,
+        })
+      }
+    }
+
```
Comment (Publicker) [inline packages/components/src/utils/parsers.ts:389]: I was just looking at this and noticed a few too many `ifs` - would you mind moving this to a `switch` structure?

Comment (arifmatin) [reply packages/components/src/utils/parsers.ts:389]: Don't know if can use switch here as we're looking for payment attempt fields here.

Comment (Publicker) [reply packages/components/src/utils/parsers.ts:389]: You're right! Sorry about that.

**Review by Publicker — APPROVED** [review]

**Pull request merged**